.changeset/polite-bags-collect.md

```diff
@@ -0,0 +1,7 @@
+---
+'@commercetools-uikit/search-select-input': minor
+'@commercetools-uikit/async-select-input': minor
+'@commercetools-uikit/select-input': minor
+---
+
+Added menuIsOpen property to SelectInputs
```

Comment:
I guess it would help if we explain a little bit the new property:

```suggestion
We introduced a new (_boolean_) property (**menuIsOpen**) to Select Inputs which allows to render the input with the dropdown initially unfolded.
```

Reply:
I would describe it more as a way to enforce the dropdown to be opened. As you can open it programatically even after initialization. Wdyt?